.github/workflows/ci.yml

```diff
@@ -376,14 +376,14 @@ jobs:
           docker exec sentry_snuba snuba migrations migrate --force
 
       - name: Run snuba tests
-        if: needs.files-changed.outputs.datasets == 'false'
+        if: needs.files-changed.outputs.api_changes == 'false'
         working-directory: sentry
         run: |
           make test-snuba
 
       - name: Run full tests
         # TODO: Adjust which changed files we care about
```

Comment:
You can remove this line

Reply:
because it's the default behavior?

Reply:
It was intented as note to myself to confirm with search team which files we want to run the full tests on in addition to the *.yamls. I think this PR does that now.